ADAL/src/public/ADAuthenticationContext.h

```diff
@@ -240,7 +240,7 @@ typedef enum
 
 /*!
  */
-+ (BOOL)handleBrokerResponse:(nonnull NSURL*)response;
++ (BOOL)handleBrokerResponse:(nonnull NSURL*)response sourceApplication:(nullable NSString *)sourceApplication;
```

Comment:
why did we have this as public if we did swizzling?

Reply:
I checked the history a little bit, seems at the time when this API was added, there was no swizzling in the code: https://github.com/AzureAD/azure-activedirectory-library-for-objc/commit/3d213af209a74c9beb62a6239f867740d55ff585#diff-f642b51cf84f1fdda84fd84cff8c5b8b

So I also think it is no longer needed today :)

Reply:
Can we deprecate it? It seems it wouldn't do anything useful today, because ADAL would swizzle first.

Reply:
addressed by moving it to internal header